sonar-project.properties

```diff
@@ -1,16 +1,5 @@
-sonar.sources=src/app
-sonar.test=src/app
+sonar.sources=projects/metis/src/app,projects/sandbox/src/app,projects/shared/src/lib
 sonar.test.inclusions=**/*.spec.ts
-sonar.typescript.lcov.reportPaths=coverage/lcov.info
-
-sonar.issue.ignore.multicriteria=e1,e2,e3
-# prettier adds leading zeros
-sonar.issue.ignore.multicriteria.e1.ruleKey=scss:leading-zeros
-sonar.issue.ignore.multicriteria.e1.resourceKey=**/*.scss
-# for angular app-etc selectors
-sonar.issue.ignore.multicriteria.e2.ruleKey=scss:unknown-type-selector
-sonar.issue.ignore.multicriteria.e2.resourceKey=**/*.scss
-# for angular :host selectors
-sonar.issue.ignore.multicriteria.e3.ruleKey=scss:unknown-pseudo
-sonar.issue.ignore.multicriteria.e3.resourceKey=**/*.scss
-
+sonar.exclusions=**/_mocked/**.ts
+sonar.projectKey=Metis-UI
+sonar.javascript.lcov.reportPaths=projects/metis/coverage/Chrome\ Headless\ 89.0.4389.90\ \(Linux\ x86_64\)/lcov.info,projects/sandbox/coverage/Chrome\ Headless\ 89.0.4389.90\ \(Linux\ x86_64\)/lcov.info,projects/shared/coverage/Chrome\ Headless\ 89.0.4389.90\ \(Linux\ x86_64\)/lcov.info
```

Comment:
What's this number? 89.0.4389.90? It really looks like a Chrome version number. I don't think that's sustainable to have that hardcoded, as it will be outdated in a matter of weeks. In fact, as we speak, I think my chrome is currently at 89.0.4389.114 already.

Reply:
Yes, that's what it is.  The numbers will change - I included this as an indicator for developers (this file is used by the developer who wants to run a local scan on the project - not by any build) but the developers have access to coverage information elsewhere.  For those reasons I've removed it.